includes/admin/tools.php

```diff
@@ -1115,10 +1115,12 @@ function edd_tools_sysinfo_get() {
 
 	$return  = apply_filters( 'edd_sysinfo_after_user_browser', $return );
 
+	$locale = get_locale();
+
 	// WordPress configuration
 	$return .= "\n" . '-- WordPress Configuration' . "\n\n";
 	$return .= 'Version:                  ' . get_bloginfo( 'version' ) . "\n";
-	$return .= 'Language:                 ' . ( defined( 'WPLANG' ) && WPLANG ? WPLANG : 'en_US' ) . "\n";
+	$return .= 'Language:                 ' . ( !empty( $locale ) && $locale ? $locale : 'en_US' ) . "\n";
```

Comment:
This can be simplified a little to just  `! empty( $locale ) ? $locale : 'en_US' . "\n";`